src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,53 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
-    public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+    private static final int INITIALIZOR = 0;
```

Comment:
No need to have constant for your "for-loop". It makes your code difficult to read. 
It's ok to use:
for (int i = 0; i < names.length; i++)
```suggestion
```